addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/aws-cfn-service.js

```diff
@@ -79,6 +135,77 @@ class AwsCfnService extends Service {
     return permissionsTemplateRaw.TemplateBody;
   }
 
+  // This will need to be wrapped into aws-accounts-service once edit functionality is merged
+  async getAndUploadTemplateForAccount(requestContext, accountId) {
+    await this.assertAuthorized(
+      requestContext,
+      { action: 'get-upload-cfn-template', conditions: [allowIfActive, allowIfAdmin] },
+      { accountId },
+    );
+    const cfnTemplateInfo = {};
+    const createParams = {};
+
+    const awsAccountsService = await this.service('awsAccountsService');
+    const cfnTemplateService = await this.service('cfnTemplateService');
+    const s3Service = await this.service('s3Service');
+
+    const account = await awsAccountsService.mustFind(requestContext, { id: accountId });
+    cfnTemplateInfo.template = await cfnTemplateService.getTemplate('onboard-account');
+    cfnTemplateInfo.region = this.settings.get(settingKeys.awsRegion);
+    cfnTemplateInfo.name = account.cfnStackName || ['SWB-Onboard-', randomString(20)].join('');
```

Comment:
The naming convention used for the create member account workflow is initial-stack-<timestamp>, e.g. initial-stack-1594674563467. Maybe keep using it to make things easier?  https://github.com/awslabs/service-workbench-on-aws/search?q=initial-stack-

Reply:
Sure, I can do that--I just need to change the account permissions yml too then

Reply:
this is going to throw off a lot of my testing longterm--I've made a note and I'll make this change in the next PR, if that's okay